qualtran/bloqs/arithmetic/bitwise_test.py

```diff
@@ -36,3 +40,28 @@ def test_xork_classical_sim():
         ctrl_out, x_out = cbloq.call_classically(ctrl=1, x=x)
         assert ctrl_out == 1
         assert x_out == x ^ k
+
+
+def test_xor(bloq_autotester):
+    bloq_autotester(_xor)
+
+
+def test_xor_symb(bloq_autotester):
+    bloq_autotester(_xor_symb)
+
+
+def test_xor_call():
+    bloq = _xor()
+    x, y = bloq.call_classically(x=7, y=0)
+    assert x == 7 and y == 7
+    x, y = bloq.decompose_bloq().call_classically(x=7, y=0)
+    assert x == 7 and y == 7
```

Comment:
Please add classical action tests when `y != 0`  so the xoring behavior is verified. Probably best to iterate over all input values and verify the classical action.